### ISSUE TITLE
Minor bug in GIFs

### ISSUE THREAD

**Issue by kodonnell** [swils/vscode-markdown-scripture#9]
FYI the GIFs show e.g. `jn#2:1-3` but these don't work out-of-box currently - they need to be `john` not `jn`. I spent a wee while debugging this before figuring it out by accident.

**Comment by swils** [comment]
Yes, this is true. For the GIF, I didn't follow my own proposed names from [SOURCES.md](https://github.com/swils/vscode-markdown-scripture/blob/master/SOURCES.md). I will fix this.

Looking forward, the issue is greater than the GIFs. It's related to UX and user comfort. I want to allow users to customise this (some people do prefer "jn" over "john" or "1kn" over "1kings") and perhaps provide autocompletion so users don't have to guess.

As a side question, do you use the included `en/drc` source? (In order to better prepare for improvements, I wonder how people are using this extension...)

**Comment by kodonnell** [comment]
Thanks - regarding more than GIFs, see my latest issue = ) #10 

I did use `en/drc` purely as it worked out of box (once I added it to the workspace settings). Realistically I'd want to use the ESV if possible (copyright etc.). I'm not using this extension at this stage - I might, but see #10 ... it doesn't quite tick all the boxes for me, the main one being backlinks. It's not a show-stopper, as I could probably script something to migrate in future - but still I'm keen to start with the "right" tool.
